sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -0,0 +1,73 @@
+# Azure SDK Dynamic JSON samples
+
+Azure SDK client [Protocol methods](ProtocolMethods.md) do not take or return model types.  JSON response content can be accessed using BCL JSON types, but use of these APIs can result in code that is difficult to read and obscures the author's intent.  To improve the developer experience, Azure.Core provides a dynamic layer over JSON APIs.
+
+## Accessing Response Content
+
+### Get `dynamic` from response
+
+Dynamic content is obtained from the `Response` return value.
+
+```C# Snippet:GetDynamicJson
```

Comment:
These have to be unique throughout the repo. Maybe add "AzureCore" to namespace these e.g., "AzureCoreGetDynamicJson"? I could see "GetDynamicJson" being vague enough to yield future collisions and using your package name (or would-be package name) is a common solution.